tools/tpm2_nvread.c

```diff
@@ -133,7 +134,7 @@ static bool nv_read(TSS2_SYS_CONTEXT *sapi_context) {
         return false;
     }
 
-
+    FILE *outputFp = NULL;
```

Comment:
nO cAmEl cAsing

Read: misc/coding_standard_c.txt

Also, variables should be declared at use